runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -900,4 +972,35 @@ private String capsString(String str)
 	{
 		return str.substring(0, 1).toUpperCase() + str.substring(1);
 	}
+
+	// check if player is carrying the item
+	private boolean itemOnPlayer(int itemID)
+	{
+		return checkItemContainer(InventoryID.INVENTORY, itemID) || checkItemContainer(InventoryID.EQUIPMENT, itemID);
+	}
+
+	private boolean checkItemContainer(InventoryID id, int itemID)
+	{
+		final ItemContainer itemContainer = client.getItemContainer(id);
+		// can item be substituted with slayer helmet?
```

Comment:
If additional comments are needed to explain what `SLAYER_HELM_ITEMS` is for, that comment should be on the list definition, not this particular line of code.